src/components/mainLayout/footer/footer.tsx

```diff
@@ -2,29 +2,22 @@
 
 import Link from "next/link";
 
-import { useWindowWidth } from "@react-hook/window-size";
-
-import cn from "classnames";
-
-import { usePathname } from "next/navigation";
-
 import { MainPath } from "../../../services/router/routes";
 
 import styles from "./footer.module.scss";
 
-const Footer = () => {
-	const pathname = usePathname();
-	const currentWindowWidth = useWindowWidth();
-	const windowWidth = 1024;
+interface IFooterProps {
+	isMainPage?: boolean;
+}
+
+const Footer = ({ isMainPage }: IFooterProps) => {
 	const actualDate: Date = new Date();
 	const copyrightYear: number = actualDate.getFullYear();
+	console.log(isMainPage);
 
 	return (
 		<div className={styles.footerWrap}>
-			<div
-				className={cn(styles.footerContainer, {
-					[styles.footerContainer__AboutAppPage]: pathname === MainPath.AboutApp && currentWindowWidth > windowWidth,
-				})}>
+			<div className={`${styles.footerContainer} ${isMainPage ? styles.footerContainer__mainPage : ""}`}>
```

Comment:
className={cn(styles.footerContainer, {[styles.footerContainer__mainPage]: isMainPage})}

we use classnames library for beautify css class expression like these